packages/string-templates/test/helpers.spec.ts

```diff
@@ -483,3 +483,26 @@ describe("uuid", () => {
     expect(output).toMatch(UUID_REGEX)
   })
 })
+
+describe("helper overlap", () => {
```

Comment:
Would be good to have some tests here that pass in `noHelpers` to make sure all those conditionals work correctly.

Also do any of these tests exercise `prefixStrings`? I can't quite tell.

Reply:
Yeap - these tests specifically have to exercise it - `sum` and `test` are helpers, so for their values to be returned they have to be prefixed.